.gitignore

```diff
@@ -6,3 +6,4 @@
 /null/
 /project/project/target/
 /.sbt/repositories
+/cmdLine/test
```

Comment:
what is this?

Reply:
was creating the tests for cmdLine, but I decided to ignore those since those aren't quite ready.